packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -4248,6 +4248,101 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/reject:
+    post:
+      tags:
+        - eservices
+      summary: reject a delegated new e-service version
+      operationId: rejectDelegatedEServiceDescriptor
+      description: Rejects a delegated E-Service descriptor
+      requestBody:
+        description: Payload containing the reason of the rejection
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RejectDelegatedEServiceDescriptorSeed"
+        required: true
+      parameters:
+        - name: eserviceId
+          in: path
+          description: The E-Service ID of the Descriptor
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: descriptorId
+          in: path
+          description: The ID of the delegated Descriptor to reject
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: New delegated e-service version rejected
```

Consistency with the rest of the M2M spec
```suggestion
          description: E-Service Descriptor rejected
```